src/Mailer/NewsletterNotification.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+namespace App\Mailer;
+
+use Symfony\Component\Mailer\MailerInterface;
+use Symfony\Component\Mime\Address;
+use Symfony\Component\Mime\Email;
+
+class NewsletterNotification
+{
+    private MailerInterface $mailer;
+
+    public function __construct(MailerInterface $mailer)
+    {
+        $this->mailer = $mailer;
+    }
+
+    public function sendEmailNotification(string $email): void
+    {
+        $emailToSend = (new Email())
+            ->from('comanpatricia27@gmail.com')     //de sters dupa ce-mi aduc main-ul la zi
```

Comment:
Remove the comment

Reply:
I deleted the entire row because of the setting is created in mailer.yaml